src/documentation/pages/components/footer/examples/_internal-brightspeed.vue

```diff
@@ -0,0 +1,182 @@
+                li
+                  a(href="https://www.lumen.com/help/en-us/home.html" target="_blank") Support
+                li
+                  a(href="https://www.lumen.com/en-us/about/legal/commission-for-complaints-for-telecom-television-services-ccts") Canadian Ombudsman
+                li
+                  a(href="https://assets.lumen.com/is/content/Lumen/gender-pay-gap-report?Creativeid=b565bf28-6d55-4f86-a245-2f0eb40b91a3" target="_blank") Gender Pay Gap 2020 (UK)
+                li
+                  a(href="https://www.lumen.com/es-ar/about/legal/latam-tariffs-regulatory.html") Legal (Latin America)
+                li
+                  a(href="https://www.lumen.com/en-us/about/legal/de-legal-notices.html") Legal Notices (Germany)
+                li
+                  a(href="https://assets.lumen.com/is/content/Lumen/modern-slavery-statement-1?Creativeid=5dbb9687-e4a8-4a40-ae00-673ce51d6a80" target="_blank") Modern Slavery Statement 2020 (UK)
+                li
+                  a(href="https://www.lumen.com/en-us/about/legal/uk-tax-strategy.html") UK Tax Strategy
+              .chi-footer__copyright
+                | &copy; 2022 Lumen Technologies. All Rights Reserved.
+                | Lumen is a registered trademark in the United States, EU and certain other countries.
+    <pre class="language-html" slot="code-webcomponent">
+      <code v-highlight="$data.codeSnippets.webcomponent" class="html"></code>
+    </pre>
+    <Wrapper slot="code-htmlblueprint">
+      <JSNeeded />
+      <pre class="language-html">
+        <code v-highlight="$data.codeSnippets.htmlblueprint" class="html"></code>
+      </pre>
+    </Wrapper>
+  </ComponentExample>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+
+declare const chi: any;
+
+@Component({
+  data: () => {
+    return {
+      exampleTabs: [
+        {
+          disabled: true,
+          id: 'webcomponent',
+          label: 'Web component'
+        },
+        {
+          active: true,
+          id: 'htmlblueprint',
+          label: 'HTML blueprint'
+        }
+      ],
+      codeSnippets: {
+        webcomponent: ``,
+        htmlblueprint: `<footer class="chi-footer">
+  <div class="chi-footer__content">
+    <div class="chi-footer__internal">
+      <div class="chi-footer__internal-content -mw--1200">
+        <div class="chi-dropdown chi-footer__language">
+          <a class="chi-button -icon -flat -light -sm chi-dropdown__trigger" id="language-dropdown-button" data-position="top-start" aria-label="Select your preferred language">
+            <div class="chi-button__content">
+              <i class="chi-icon icon-globe-network" aria-hidden="true"></i>
+              <span>English</span>
+            </div>
+          </a>
+          <div class="chi-dropdown__menu -w--sm -text--body">
```

Comment:
The same array should be reused here and in all the code snippets as in case of Picker Group component.